src/feature.cpp

```diff
@@ -700,8 +700,47 @@ void Feature::Triangulate(const SE3 &gsb, const SE3 &gbc,
   Vec2 xc1 = CameraManager::instance()->UnProject(front());
   Vec2 xc2 = CameraManager::instance()->UnProject(back());
   SE3 g12 = (ref_->gsb() * gbc).inv() * (gsb * gbc);
-  Vec3 Xc1 = options.method == 1 ? Triangulate1(g12, xc1, xc2)
-                                 : Triangulate2(g12, xc1, xc2);
+
+  Vec3 Xc1;
```

Comment:
XIVO uses 1->2 convention for triangulation. The paper uses 0->1 convention. Since we are in `xivo::Feature::Triangulate` and not the function implementing the paper, we should use 1->convention throughout here --- variable should be named `Xc2`.